entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
```

Comment:
Shell scripting best practice: `set -euo pipefail` 
https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425

Reply:
Tried it got a warning, apparently it doesn't work if we don't use [bash](https://github.com/koalaman/shellcheck/wiki/SC3040). Also since our script is very minimal I don't think we really need it.